packages/bpk-component-card-list/src/BpkCardListRail/BpkCardListRail.tsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-undef */
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { ReactElement } from 'react';
+
+import BpkMobileScrollContainer from '../../../bpk-component-mobile-scroll-container';
+import { cssModules } from '../../../bpk-react-utils';
+
+import STYLES from './BpkCardListRail.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type BpkCardListRailProps = {
+  children: ReactElement[];
+};
+
+const BpkCardListRail = ({ children }: BpkCardListRailProps) => (
+  <div className={getClassName('bpk-card-list--rail_scrollContainer')}>
+    <BpkMobileScrollContainer>
+      <div className={getClassName('bpk-card-list--rail_container')}>
+        {children.map((item: JSX.Element) => (
+          <div
+            key={item.key}
+            className={getClassName('bpk-card-list--rail_card')}
+          >
+            {item}
+          </div>
+        ))}
+      </div>
+    </BpkMobileScrollContainer>
```

Comment:
I've been looking over the designs and implementation in storybook for this rail component, and there seems like the "peek" of the next card in the list is missing:




By looking in Figma it seems like this is a necessary feature so I wondered if the implementation caters for this? or if there are plans to add this at some point in the future?

| figma | storybook |
|----|----|
| ![image](https://github.com/Skyscanner/backpack/assets/1532576/b3e918db-2b61-4ad8-b3c8-96c74edc4765)  |  ![image](https://github.com/Skyscanner/backpack/assets/1532576/b8ec069f-5089-4943-88b0-8a5e2dc96c8c)